RELEASE/scripts/autoscend/combat/auto_combat_default_stage4.ash

```diff
@@ -235,23 +235,23 @@ string auto_combatDefaultStage4(int round, monster enemy, string text)
 	}
 	
 	//this completes the quest Advertise for the Mysterious Island Arena which is a sidequest which accelerates the L12 frat-hippy war quest
-	if((canUse($item[Rock Band Flyers]) || canUse($item[Jam Band Flyers])) && (my_location() != $location[The Battlefield (Frat Uniform)]) && (my_location() != $location[The Battlefield (Hippy Uniform)]) && !get_property("auto_ignoreFlyer").to_boolean())
+	if((canUse($item[Rock Band Flyers]) || canUse($item[Jam Band Flyers])) && (get_property("flyeredML").to_int() < 10000) && (my_location() != $location[The Battlefield (Frat Uniform)]) && (my_location() != $location[The Battlefield (Hippy Uniform)]) && !get_property("auto_ignoreFlyer").to_boolean())
```

Comment:
`get_property("flyeredML").to_int() < 10000)`
IIRC we do not use this on purpose as sometimes mafia tracks this wrong...
but... also IIRC we have an automatic fix in place where if mafia thinks it is >= 10000 and it fails to finish the quest we set it back to 9999
so this is probably fine as is.

Reply:
if you look below, both checks for rock band flyers were checking flyeredML I just moved the check up here instead.

Reply:
I see now. I missed that. thanks for pointing it out